hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -132,6 +138,8 @@ public class ObjectEndpoint extends EndpointBase {
 
   private List<String> customizableGetHeaders = new ArrayList<>();
   private int bufferSize;
+  private int chunkSize;
+  private boolean datastreamEnabled;
```

Comment:
Please make the new fields final.

Reply:
Indeed, we should create an `ObjectEndpointStreaming streaming` object and use streaming != null for checking if streaming is enabled.